src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -26,13 +29,46 @@ export default function VitalPercents(props: Props) {
         return (
           <VitalStatus key={p.vitalState}>
             {p.vitalState === VitalState.POOR && (
-              <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Poor'
+                    : tct('Poor: >[threshold]ms', {
+                        threshold: webVitalPoor[props.vital],
+                      })
+                }
+              >
+                <IconFire color={vitalStateColors[p.vitalState] as Color} />
+              </Tooltip>
             )}
             {p.vitalState === VitalState.MEH && (
-              <IconWarning color={vitalStateColors[p.vitalState] as Color} />
+              <Tooltip
+                title={
+                  Array.isArray(props.vital)
+                    ? 'Needs improvement'
```

Comment:
```suggestion
                    ? t('Needs improvement')
```
Display text should be in t() so we could translate it in the future.